chromadb/ingest/__init__.py

```diff
@@ -77,12 +81,13 @@ class Consumer(Component):
     def subscribe(
         self,
         collection_id: UUID,
+        segment_id: UUID,
```

Comment:
the `segment_id` is needed here so that `.ack()` can take a `subscription_id`
this feels weird as Producers don't need to know about `segment_id`s outside of this
one alternative is providing the `segment_id` to `ack()` instead: 11566569
another is making `ack()` a utility instead